test/iron-dropdown.html

```diff
@@ -146,11 +171,14 @@
           dropdown = fixture('NonLockingDropdown');
         });
 
-        test('can be disabled with `allowOutsideScroll`', function() {
+        test('can be disabled with `allowOutsideScroll`', function(done) {
           dropdown.open();
 
-          expect(Polymer.IronDropdownScrollManager.elementIsScrollLocked(document.body))
-            .to.be.equal(false);
+          Polymer.Base.async(function() {
```

Comment:
Why is async required here? Is `document.body` scroll locked for some temporary period of time? Or are you waiting for the dropdown to be fully opened to make sure it doesn't get scroll locked?